salt/edx/templates/ansible_env_config.yml.j2

```diff
@@ -115,8 +115,8 @@ lms_auth_config:  &lms_auth
       password: "{{ MONGODB_CONTENTSTORE_PASSWORD }}"
       port: "{{ MONGODB_PORT }}"
       user: "{{ MONGODB_CONTENTSTORE_USER }}"
-    ADDITIONAL_OPTIONS:
```

Comment:
Digging a bit deeper it appears as though `ADDITIONAL_OPTIONS` just gets merged back into `OPTIONS` anyway. https://github.com/mitodl/edx-platform/blob/mitx/dogwood.3-1rc1/common/lib/xmodule/xmodule/contentstore/django.py#L27